.travis.yml

```diff
@@ -69,14 +64,13 @@ before_script:
 script:
 - ./tools/lint.py
 - bash -c "sleep 2100; pkill ninja" &
-- TIMEFORMAT="%E" # set time format
-- TRAVIS_BUILD_DURATION=$(time ./tools/build.py -j2 3>&1 1>&2 2>&3) # swap stdout and stderr
-- TEST_DURATION=$(time ./tools/test.py $DENO_BUILD_PATH 3>&1 1>&2 2>&3) # swap stdout and stderr
+- BUILD_DURATION=$(/usr/bin/time -p ./tools/build.py -j2 3>&1 1>&2 2>&3 | awk '/real/ {print $2}') # swap stdout and stderr
```

Comment:
That's a pretty mind boggling concoction. Did you consider using tail and using a temp file?